### PR TITLE
make tests less flaky

### PR DESCRIPTION
**Pull request by kostekIV**

### CODE REVIEW

File: local-tests/test_catch_up.py [hunk 1]
```diff
@@ -36,29 +36,40 @@
...
+finalized_before_start_per_node = check_finalized(chain)
+
+# Check if the finalization didnt stopped after kill
+if finalized_before_start_per_node[0] - finalized_before_kill_per_node[0] < 10:
+    print('Finalization stalled')
+    sys.exit(1)
+
 chain.start('aleph', nodes=[3, 4])
-check_finalized(chain)
+finalized_after_start = check_finalized(chain)
```
Comment (h4nsu) [inline local-tests/test_catch_up.py:57]: You don't seem to be using this variable

File: local-tests/test_catch_up.py [hunk 2]
```diff
@@ -36,29 +36,40 @@
...
+
 print('waiting 45s for catch up')
 sleep(45)
-finalized_per_node = check_finalized(chain)
+finalized_after_catch_up_per_node = check_finalized(chain)
+
+nonvalidator_diff = finalized_after_catch_up_per_node[4] - finalized_before_start_per_node[0]
```
Comment (h4nsu) [inline local-tests/test_catch_up.py:63]: Is there a reason to compare that between different nodes? Why not
`finalized_after_catch_up_per_node[4] - finalized_before_start_per_node[0]`?
Same applies to the line below

**Review by h4nsu — APPROVED** [review]
LGTM

File: local-tests/test_catch_up.py (hunk 2 above)
Comment (kostekIV) [reply local-tests/test_catch_up.py:63]: did you meant `finalized_after_catch_up_per_node[4] - finalized_before_start_per_node[4]`?

Comment (kostekIV) [reply local-tests/test_catch_up.py]: done

File: local-tests/test_catch_up.py (hunk 1 above)
Comment (kostekIV) [reply local-tests/test_catch_up.py]: thx, removed

**Review by maciejzelaszczyk — APPROVED** [review]
One nit about a comment. Non-blocking. Good job :)

**Pull request merged**